main.scss

```diff
@@ -20,3 +20,7 @@
 @import "src/scss/field-errors";
 @import "src/scss/field-valid";
 @import "src/scss/global-and-section-messages";
+
+// Set silent mode back on again to avoid it being output twice
+$o-forms-is-silent: true;
+$o-ft-forms-is-silent: true;
```

Comment:
This is contra spec: http://origami.ft.com/docs/syntax/scss/#sass-variables.  The reason for the rule was because setting the values of config vars in a module creates potentially unpredictable, load-order-specific behaviour.